cmd/read.go

```diff
@@ -52,10 +54,40 @@ If --pcrs is not provided, all pcrs are read for that hash algorithm.`,
 	},
 }
 
+var nvReadCmd = &cobra.Command{
+	Use:   "nvdata <endorsement | owner | platform | null>",
+	Short: "Read NVData from the TPM",
+	Long: `Read NVData from the TPM
+
+Based on the given hierarchy, read the contents of the TPM's NVData.
+An empty password will be passed in, and the size will be the default
+to TPM_PT_NV_BUFFER_MAX`,
+	ValidArgs: getKeys(hierarchyNames),
+	Args:      cobra.ExactValidArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		rwc, err := openTpm()
+		if err != nil {
+			return err
+		}
+		defer rwc.Close()
+
+		data, err := tpm2.NVReadEx(rwc, tpmutil.Handle(nvIndex), hierarchyNames[args[0]], "", 0)
+		if err != nil {
+			return err
+		}
+		fmt.Fprintln(dataOutput(), string(data))
```

Comment:
We should just do:

```go
if _, err := dataOutput().Write(data); err != nil {
        return fmt.Errorf("outputting NVData: %v", err)
}
```

like we do in `seal`.

Reply:
done